packages/react/src/components/Details/Details.mdx

```diff
@@ -0,0 +1,121 @@
+- Ikke legg en `Details` inni en annen.
+
+<br />
+
+<Primary />
+<Controls />
+
+## Bruk
+
+```tsx
+import { Details } from '@digdir/designsystemet-react';
+
+<Details>
+  <Details.Summary>Details heading text</Details.Summary>
+  <Details.Content>Details content</Details.Content>
+</Details>
+```
+
+## Eksempler
+
+<br />
+### Med ramme
+
+`Details` kan vises med ramme. Dette kan passe i tilfeller der detailss ikke fyller hele siden, eller når det kun er en rad.
+
+<Canvas of={DetailsStories.InCard} />
+
+### Med farger
+
+`Details` kan vises i farger fra ditt brand.
+
+<Canvas of={DetailsStories.InCardWithColor} />
+
+#### Kontrollert
+
+`Details` har egen tilstand på åpen/lukker, men dette kan kontrolleres utenfra.
+
+<Canvas of={DetailsStories.Controlled} />
+
+<br />
+
+## Retningslinjer
+
+Tester viser at brukerne sjeldnere ser på skjult innhold, enn det som er synlig direkte på siden. Ikke bruk `Details` til å skjule innhold for å gjøre siden "ryddigere". Finn ut om du faktisk må skjule innhold og vær klar over hvorfor du gjør det. Tenk over om det er lurt å vise det viktigste innholdet i åpen status når brukeren kommer inn på siden.
+
+Hvis innholdet er for langt eller komplisert, bør du heller omformulere teksten og/eller eventuelt fordele den på flere sider.
+
+### Egnet til
+
+- samle innhold som er litt lengre
+- gjøre det frivillig å se innhold som er litt mindre viktig enn alltid synlig innhold
+- vise ofte stilte spørsmål
+- vise tilleggsinformasjon som kan være til hjelp for brukerne
+
+### Ikke egnet til
+
+- vise små mengder innhold
+- vise informasjon hvis det bare er ett element
+- vise viktig innhold som alle bør se når de kommer til siden (for eksempel feilmeldinger)
+- gi mer informasjon om et spørsmål i et skjema – det innholdet bør brukeren se med en gang
+- velge mellom ulike alternativer
+- skjule innhold fra søkeresultater eller oversikter/tabeller
+- dele opp en logisk flyt eller en rekke med handlinger, da bør du heller bruke en trinnvis liste
+
+### Unngå
+
+- Ikke legg en `Details` inni en annen, det vi kaller nøstede lister.
+
+<br />
+
+## Tekst i komponenten
+
+Details skal brukes til maksimalt to avsnitt innenfor hvert nedtrekk. Sørg for at overskriften gir en god beskrivelse av hva innholdet i details er. En tydelig og beskrivende overskrift skal gjøre brukerne nysgjerrige på innholdet. Overskriftene til details kan ha stor betydning for om brukerne finner det de trenger, om innholdet blir lest og om det kan regnes som tilgjengelig for alle brukere. «Vis mer» eller «Les mer her» er ikke gode nok titler. Har du en details med mange nedtrekk, kan du ha en hovedoverskrift eller temaoverskrift over hele listen.
+
+<br />
+
+## Tilgjengelighet
+
+Som standard er `Details.Summary` et `h1` element. Du må selv velge riktig overskriftsnivå for din side. Hvis `Details.Summary` er en del av en større overskrift, kan du bruke `h2`, `h3` osv. for å opprettholde en logisk rekkefølge.
+
```

Comment:
Just noticed that we still have this, might as well remove it now as well?

Reply:
oh, true!